### DISCUSSION TITLE
Uptime history 404

### DISCUSSION THREAD

**Discussion by LaMpiR** [upptime/upptime#233] [Q&A] [answered]
Hi, so I have started a small test to check it out since it looks amazing. Any help would be appreciated. 

https://github.com/LaMpiR/uptime

I followed the guidelines, but when I click on the uptime history, I get:

```
404
File not found

The site configured at this address does not contain the requested file.

If this is your site, make sure that the filename case matches the URL.
For root URLs (like http://example.com/) you must provide an index.html file.
```

**Comment by AnandChowdhary** [comment, marked as answer]
You have to set gh-pages as the branch in your GitHub Pages settings: https://upptime.js.org/docs/get-started#enable-publishing

**Reply by LaMpiR** [reply]
Thank you :) I thought that already did that.
